packages/jest-snapshot/src/SnapshotResolver.ts

```diff
@@ -97,7 +97,7 @@ function mustImplement(propName: string, requiredType: string) {
     chalk.bold(
       `Custom snapshot resolver must implement a \`${propName}\` as a ${requiredType}.`,
     ) +
-    '\nDocumentation: https://facebook.github.io/jest/docs/en/configuration.html#snapshotResolver'
+    '\nDocumentation: https://facebook.github.io/jest/docs/configuration.html#snapshotResolver'
```

Comment:
hey! this looks like .html should be removed here :)

Reply:
Bah, I thought I got them all 😅 Good catch!